frame/example/src/lib.rs

```diff
@@ -785,4 +824,62 @@ mod tests {
 		let info = custom_call.get_dispatch_info();
 		assert_eq!(info.weight, 2000);
 	}
+
+	#[test]
```

Comment:
This test is weird to have in this crate.

You should keep tests for the macro with the macro code imo.

Reply:
Yeah, I was hesitating between writing it here or in the `benchmarking` crate. I choose here so I don't have to write an `example` code. But I can move if you think it's better.